gearbox.go

```diff
@@ -151,6 +151,7 @@ func (gb *gearbox) newHTTPServer() *fasthttp.Server {
 
 // Stop serving
 func (gb *gearbox) Stop() error {
+	fmt.Printf("Service stopped listening to %s", gb.httpServer.Name)
```

Comment:
What about
>  gearbox v... stopped listening on %s 
(use gearbox name and version from constants)

Reply:
Okay.